src/lib.rs

```diff
@@ -86,6 +116,30 @@ pub fn initialize(_argc: *const isize, _argv: *const *const *const u8) -> isize
 /// # mod my_crate { pub fn parse(_: &[u8]) -> Result<(), ()> { unimplemented!() } }
 /// ```
 ///
+/// ## Rejecting Inputs
+///
+/// To indicate whether an input should be kept in or rejected from the corpus,
+/// return a [Corpus] value from your fuzz target. For example:
+///
+/// ```no_run
+/// #![no_main]
+///
+/// use libfuzzer_sys::{Corpus, fuzz_target};
+///
+/// fuzz_target!(|input: String| -> Corpus {
+///     let parts: Vec<&str> = input.splitn(2, '=').collect();
+///     if parts.len() != 2 {
+///         return Corpus::Reject;
+///     }
+///
+///     let key = parts[0];
+///     let value = parts[1];
+///     my_crate::parse(key, value);
```

Comment:
```suggestion
///     let _ = my_crate::parse(key, value);
```

Reply:
To clarify: this change is just to highlight to readers that we are ignoring the results of the parse, and are just calling this function to fuzz it.

Reply:
I've updated this to match the other code example (`let _result: Result<_, _> =`) so it's obvious these are the same line.